generators/bdk/python/index.js

```diff
@@ -0,0 +1,62 @@
+const Generator = require('yeoman-generator')
```

Comment:
If I'm bot wrong in JS we need the `;` at each line, that is different from Python :)

Reply:
not really convinced about this one but I will update it anyway :))

Reply:
; is not mandatory in JS
but we could probably solve this by using eslint on the project